tests/runtime/conftest.py

```diff
@@ -23,7 +23,21 @@ def print_method_name(request):
 
 @pytest.fixture
 def temp_dir(tmp_path_factory: TempPathFactory) -> str:
-    return str(tmp_path_factory.mktemp('test_runtime'))
+    """
+    Creates a unique temporary directory
+
+    Parameters:
+    - tmp_path_factory (TempPathFactory): A TempPathFactory class
+
+    Returns:
+    - str: The temporary directory path that was created
+    """
+
+    unique_suffix = int(time.time() * 1000)
```

Comment:
I'd suggest a bit cleaner variation for this (add `import random` to the imports list):
```
    unique_suffix = random.randint(10000, 99999)
```
This will give you a random 5-digit number.

Reply:
Updated